src/character.cpp

```diff
@@ -248,26 +248,22 @@ void Character::recalc_sight_limits()
         // Yes, I'm breaking the cap. I doubt the reality bubble shrinks at night.
         // BIRD_EYE represents excellent fine-detail vision so I think it works.
         if (has_trait("BIRD_EYE")) {
-            sight_boost = 13;
+            sight_boost_cap = 13;
```

Comment:
The else is gone.
I wasn't totally sure about this, was it supposed to set the boost and the cap to 13 before if the player has the BIRD_EYE trait, or set the boost to 13 and leave the cap alone?  Previously it was adjusting the boost, but leaving the cap alone, which AFAICT means it's effectively never going to happen, because when it's applied it does this:
```
if (light_level < sight_boost_cap) {
    light_level = std::min(light_level + sight_boost, sight_boost_cap);
}
```
Now it will set both to 13, which will have an impact.